Tests/CatalogItems/Rest/Get-RsRestCacheRefreshPlanHistory.Tests.ps1

```diff
@@ -61,13 +61,20 @@ Describe "Get-RsRestCacheRefreshPlanHistory" {
             $plan | Should Not BeNullOrEmpty
             $plan.LastStatus | Should Not Be 'New Scheduled Refresh Plan'
 
-            Start-Sleep -Seconds 3
+            Start-Sleep -Seconds 6
             $someHistory = Get-RsRestCacheRefreshPlanHistory -ReportPortalUri $reportPortalUri -RsReport "$($rsFolderPath)/ReportCatalog"
             @($someHistory).count | Should be 1
             
             Start-RsRestCacheRefreshPlan -ReportPortalUri $reportPortalUri -RsReport "$($rsFolderPath)/ReportCatalog"
-            Start-Sleep -Seconds 3
-            $moreHistory = Get-RsRestCacheRefreshPlanHistory -ReportPortalUri $reportPortalUri -RsReport "$($rsFolderPath)/ReportCatalog"
+            $timer= get-date
+            while ($true) {
```

Comment:
usually is better practice to have the break conditions in the loop, you could do a Do - while loop instead and add the conditions to the end

Reply:
I had tried that earlier but it didn't work.  Turns out adding an extra set of parens fixed it.